src/Microsoft.DotNet.Build.Tasks/PackageFiles/packages.Unix.config

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="dnx-mono" version="1.0.0-rc2-16128" />
```

Comment:
It actually depends, if we want to restore dnx on the corefx side or on the buildtools side. The way I have it setup right now is that we do it in corefx side so yes, this files have no use for now, but I could change it so that corefx only restores buildtools, and buildtools restores dnx.